### PR TITLE
Update wide_deep_cross_networks example with tensorflow 2.5 changes

### PR DESCRIPTION
**Pull request by mattdangerw**

### CODE REVIEW

File: examples/structured_data/wide_deep_cross_networks.py [hunk 1]
```diff
@@ -274,25 +273,25 @@ def encode_inputs(inputs, use_embedding=False):
             # Create a lookup to convert string values to an integer indices.
             # Since we are not using a mask token nor expecting any out of vocabulary
             # (oov) token, we set mask_token to None and  num_oov_indices to 0.
-            index = StringLookup(
```
Comment (fchollet) [inline examples/structured_data/wide_deep_cross_networks.py:277]: For this example, please update the intro line "Note that this example should be run with TensorFlow 2.3 or higher." as well as the "last updated" date

File: examples/structured_data/wide_deep_cross_networks.py [hunk 2]
```diff
@@ -274,25 +273,25 @@ def encode_inputs(inputs, use_embedding=False):
...
-                vocabulary=vocabulary, mask_token=None, num_oov_indices=0
+            lookup = StringLookup(
+                vocabulary=vocabulary,
+                mask_token=None,
+                num_oov_indices=0,
+                output_mode="int" if use_embedding else "binary",
```
Comment (fchollet) [inline examples/structured_data/wide_deep_cross_networks.py:280]: I assume we'll keep this at "binary" in the future (which is totally fine even after we rename it to multi_hot, we can keep binary as a first-class alias)

**Review by fchollet — COMMENTED** [review]
Thank you for the PR! After the final changes, you can add the generated files.

File: examples/structured_data/wide_deep_cross_networks.py (hunk 1 above)
Comment (mattdangerw) [reply examples/structured_data/wide_deep_cross_networks.py:277]: Done!

**Comment by mattdangerw** [comment]
Looks like UCI ML Repository is down right now, so will try later to update the generated files.

**Comment by mattdangerw** [comment]
Files added.

**Review by fchollet — APPROVED** [review]

**Pull request merged**